### PR TITLE
Http Ingestion: add the HTTP Basic Auth

### PR DESCRIPTION
**Pull request by squioc**
Document the ability to supply the intake key as HTTP Basic Auth

### CODE REVIEW

File: docs/xdr/features/collect/ingestion_methods/https.md [hunk 1]
```diff
@@ -90,10 +92,10 @@ For numerous events, you can use the alternative endpoint `/batch`. The events s
 ```python
 import requests
 
-headers = {"X-SEKOIAIO-INTAKE-KEY": "YOUR_INTAKE_KEY"}
```
Comment (PierrickV) [inline docs/xdr/features/collect/ingestion_methods/https.md:93]: Why did you remove the previous example ? we could provide 1 example for each use case

**Review by otetard — APPROVED** [review]
